Makefile

```diff
@@ -118,7 +118,7 @@ update-helm: manifests yq
 	./hack/update-helm.sh
 
 .PHONY: generate
-generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations and client-go libraries.
+generate: gomod-download controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations and client-go libraries.
```

Comment:
Is this related to the fix?

Reply:
No, but `generate` wasn't working for me without it.

Reply:
sgtm